index.js

```diff
@@ -46,13 +46,18 @@ module.exports.plugins = [
   [require("remark-lint-first-heading-level"), 1],
   [require("remark-lint-heading-style"), "atx"],
   [require("remark-lint-list-item-indent"), "space"],
+  [require("remark-lint-mdash-style"), "-"],
```

Comment:
This is currently saying "use an en dash for your em dashes" and I'm pretty opposed to that.

```suggestion
  [require("remark-lint-mdash-style"), "—"],
```

Although I think it's fine for people to use `&mdash;` instead.